### ISSUE TITLE
Add Table parameter support

### ISSUE THREAD

**Issue by BugDiver** [BugDiver/gauge-ts#4]
Add support for typed Table as a parameter to steps.

Currently, it's available as `any` type

Example:
```typescript
import { Step } from "gauge-ts";

export default class StepImplementation {
    @Step("Almost all words have vowels <wordsTable>")
    public async verifyVowelsCountInMultipleWords(wordsTable: any) {
        for (const row of wordsTable.rows) {
            equal(await this.countVowels(row.cells[0]), parseInt(row.cells[1]));
        }
    }
}
```
**Describe the solution you'd like**
```typescript
import { Step, Table } from "gauge-ts";

export default class StepImplementation {
    @Step("Almost all words have vowels <wordsTable>")
    public async verifyVowelsCountInMultipleWords(table: Table) {
        for (let row of table.getTableRows()) {
            let word:string = row.getCell("Word");
            let expectedCount = parseInt(row.getCell("Vowel Count"));
            let actualCount = countVowels(word);
            equal(expectedCount, actualCount);
        }
    }
}
```

**Comment by x80486** [comment]
Probably fixing this one could fix also #119 :thinking:

**Comment by BugDiver** [comment]
Done via 49f12730da292609833c94592c4e75a08ff48ffb
